authentik/stages/prompt/stage.py

```diff
@@ -54,7 +62,7 @@ def __init__(self, *args, **kwargs):
         stage: PromptStage = kwargs.pop("stage_instance", None)
         plan: FlowPlan = kwargs.pop("plan", None)
         request: HttpRequest = kwargs.pop("request", None)
-        user: User = kwargs.pop("user", None)
+        self.user: User = kwargs.pop("user", None)
```

Comment:
What's this for?

Reply:
Oops that is a leftover from an earlier implementation I was experimenting with. Fixed it. Thanks